index.html

```diff
@@ -33,13 +63,45 @@ <h1 class="logo-heading">Fun Bus</h1>
         </div>
       </div>
     </div>
-  </div>
+  </div>-->
   <div class="container">
-    <div class="row">
+    <div class="column">
         <div class="col-sm-12">
             <img src="img/fun-bus.jpg" class="img-fluid" alt="Responsive image">
         </div>
     </div>
+    <!--navbar-->
+    <div class="container-fluid sticky-top"></div>
```

Comment:
This div tag where you added `sticky-top` doesn't have any content between the opening `<div>` tag and the closing `</div>` tag. I'll check it off on the list, because I can see that you did try. In the future, make sure you're checking your work-in-progress frequently to ensure that the changes you made to your code are doing what you expect.